### ISSUE TITLE
IOS: Rename Interface administratively down?

### ISSUE THREAD

**Issue by bittsi** [napalm-automation/napalm-logs#250]
The implementation if an Interface goes to admin down is named [INTERFACE_DOWN](https://github.com/napalm-automation/napalm-logs/blob/develop/napalm_logs/config/ios/INTERFACE_DOWN.yml). Also the error has this name.

Wouldn't it be better if the name would be e.g. `INTERFACE_ADMIN_DOWN`, because it's related to the `admin_status`.  If an Interface changes the `oper_status` the error should be named `INTERFACE_DOWN` respectively `INTERFACE_UP`.

For EOS it is already implemented like this for the [oper_status](https://github.com/napalm-automation/napalm-logs/blob/develop/napalm_logs/config/eos/INTERFACE_DOWN.yml).

btw:
I already did a [PR](https://github.com/napalm-automation/napalm-logs/pull/249) for the implementation of the `oper_status`. If you agree on this suggestion I would rename the error and the directory to `INTERFACE_DOWN` and `INTERFACE_UP` in the PR.

**Comment by loverend** [comment]
Yes I agree that we should differentiate between admin down and oper down. I think `INTERFACE_ADMIN_DOWN` is a good name.

**Comment by bittsi** [comment]
@loverend Is it ok if I do the renaming directly within the PR [#249](https://github.com/napalm-automation/napalm-logs/pull/249)?

**Comment by loverend** [comment]
Yes that's fine. Ideally as a separate commit though.

**Comment by bittsi** [comment]
@loverend Changes done in PR [#249](https://github.com/napalm-automation/napalm-logs/pull/249). Travis passed but coveralls fails. Is there something additionally needed from my side?

**Comment by loverend** [comment]
Looks good. You can ignore the coveralls fail.

I would like to wait for approval from @mirceaulinic before merging.
